src/hotspot/share/classfile/javaClasses.cpp

```diff
@@ -1729,6 +1730,23 @@ void java_lang_Thread::set_jvmti_thread_state(oop java_thread, JvmtiThreadState*
   java_thread->address_field_put(_jvmti_thread_state_offset, (address)state);
 }
 
+int java_lang_Thread::VTMS_transition_disable_count(oop java_thread) {
+  return java_thread->int_field(_jvmti_VTMS_transition_disable_count_offset);
+}
+
+void java_lang_Thread::inc_VTMS_transition_disable_count(oop java_thread) {
+  int val = VTMS_transition_disable_count(java_thread);
+  assert(JvmtiVTMSTransition_lock->is_locked(), "Must be locked");
```

Comment:
You want to use `owned_by_self()` for these assertions - thanks.

Reply:
You are right. Fixed now.